src/routes/tekenmethodes/[slug]/stappenplan/+page.svelte

```diff
@@ -39,132 +13,90 @@
 
 <MethodsHeader {data} />
 
-
-<article class="grid">
-  <div class="flex-s">
-    <div class="tags-b">
-
-      <img class="icon" src="/icon1.svg" alt="icon" />
-      <div class="wrapper-methods">
-
-      <img class="icon" src="/images/category.svg" alt="icon" />
-
-      {#each data.methods as method}
-        {#each method.categories as category}
-          <h3 class="tag">
-            {@html category.title}
-          </h3>
+<section class="grid">
+  <div class="tags-container">
+   
+    <div class="tags-group">
+        <img class="icon" src="/images/category.svg" alt="icon" />
+        {#each data.methods as method}
+          {#each method.categories as category}
+            <p class="tag">{@html category.title}</p>
+          {/each}
         {/each}
-      {/each}
-    </div>
     </div>
 
-    <div class="tags-b">
+    <div class="tags-group">
       <img class="icon" src="/images/materials.svg" alt="icon" />
-      <h3 class="tag">
-        {#each data.methods as method}
-          {method.material}
-        {/each}
-      </h3>
+      {#each data.methods as method}
+        <p class="tag">{method.material}</p>
+      {/each}
     </div>
 
-    <div class="tags-b">
+    <div class="tags-group">
       <img class="icon" src="/images/duration.svg" alt="icon" />
```

Comment:
Is het niet beter om inline svg te gebruiken voor de icoontjes? Dat scheelt netwerk request's

Reply:
Check